e2e_tests/tests/conftest.py

```diff
@@ -110,7 +110,7 @@ def cluster_log_manager(request: SubRequest) -> Iterator[Optional[ClusterLogMana
         # Yield `None` so that pytest handles the no log manager case correctly.
         yield None
 
-    compare_stats()
+    #compare_stats()
```

Comment:
@guangqing mentioned that she needs to fix something here. Once she does I will fetch from upstream/master. https://determined-ai.slack.com/archives/CSG3W08JY/p1649877002656269?thread_ts=1649870028.978399&cid=CSG3W08JY